src/components/vr_cooperation/include/vr_cooperation/command_factory.h

```diff
@@ -30,35 +30,48 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_MOBILE_COMMAND_FACTORY_H_
-#define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_MOBILE_COMMAND_FACTORY_H_
+#ifndef SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMAND_FACTORY_H_
+#define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMAND_FACTORY_H_
 
 #include "application_manager/message.h"
 #include "utils/macro.h"
 #include "vr_cooperation/commands/command.h"
 #include "vr_cooperation/vr_module.h"
 
+namespace vr_hmi_api {
+class ServiceMessage;
+}  // namespace vr_hmi_api
+
 namespace vr_cooperation {
+class VRModule;
 
 /**
  * @brief Factory class for command creation
  */
-class MobileCommandFactory {
+class CommandFactory {
  public:
   /**
    * @brief Create command object and return pointer to it
    * @param  msg Message shared pointer.
    * @return Pointer to created command object.
    */
   static commands::Command* CreateCommand(
-      VRModule* parent,
-      const application_manager::MessagePtr& msg);
+      VRModule* parent, const application_manager::MessagePtr& msg);
+
+  /**
+   * @brief Create command object and return pointer to it
+   * @param parent pointer to VRmodule
+   * @param  message Message shared pointer.
+   * @return Pointer to created command object.
+   */
+  static commands::Command* CreateCommand(
```

Comment:
Can CommandFactory create something else excluding commands? I believe `Create` is better.